packages/foreman/foreman/foreman.spec

```diff
@@ -905,6 +907,10 @@ rm -rf ./usr \\
 %%{?-s:rm -rf %%{buildroot}%%{gem_instdir}/public/webpack/images}
 EOF
 
+%if 0%{?fedora} || 0%{?rhel} >= 8
+grep -lr '#!/usr/bin/python' usr/share/foreman/extras/noVNC | xargs sed -i 's/#!\/usr\/bin\/python/#!\/usr\/bin\/python3/g'
```

Comment:
At this point it's too late, as this file has already been copied into the buildroot. I'd move this to just after `%install` section and you can drop the `usr/share/foreman` path